interface/super/edit_layout_props.php

```diff
@@ -106,10 +106,10 @@ function submitProps() {
   <td>
    <select name='form_columns'>
 <?php
+  echo "<option value=''>" . xlt('Default') . " (4)</option>\n";
   for ($cols = 2; $cols <= 10; ++$cols) {
-    echo "<option value=";
-    echo (($cols == 4) ? "'' selected" : "'$cols'");
-    echo ">$cols</option>\n";
+  	if ($cols == 4) continue;
+    echo "<option value='$cols'>$cols</option>\n";
```

Comment:
If you want to keep the if statement then suggest

```php
if ($cols == 4) {
    echo "<option value='$cols'>$cols</option>\n";
}
```

[PSR ](http://www.php-fig.org/psr/psr-2/#if-elseif-else) requires `if` statements have curly braces, 1 line if's are not allowed